main.js

```diff
@@ -2064,7 +2104,12 @@ function processMessage(msg) {
         case 'getDiagData':
             if (msg.callback && msg.from) {
                 if (msg.message) {
-                    collectDiagInfo(msg.message, obj => sendTo(msg.from, msg.command, obj, msg.callback));
+                    try {
+                        const obj = await collectDiagInfo(msg.message);
+                        sendTo(msg.from, msg.command, obj, msg.callback);
+                    } catch {
```

Comment:
Should we print the error here?

Reply:
Until now this was not case, but we can add logging here, won’t hurt imho